plugins/cros-ec/fu-cros-ec-usb-device.c

```diff
@@ -114,6 +129,8 @@ fu_cros_ec_usb_device_probe (FuUsbDevice *device, GError **error)
 
 	/* very much like usb_updater2's usb_findit() */
 
+	g_print ("fu_cros_ec_usb_device_probe\n");
+
```

Comment:
(this is obviously a debugging line, so just a note to remove it when you no longer need it)